ItemsListApp/test/ItemsListApp.Api.UnitTests/Tests/Controllers/ItemsControllerUnitTests.cs

```diff
@@ -28,28 +29,27 @@ public void SetUp()
         }
 
         [Test]
-        public async Task TestGet()
+        public async Task Get_IdOfItem_ReturnsItemWithThisId()
         {
             var expected = new Item
             {
-                Id = Guid.NewGuid(),
+                Id = new Guid("6341EB90-93E6-49AA-BBEC-2A69C3C8DB8D"),
                 Text = "Text of required item",
             };
 
             var action = await _controller.GetAsync(expected.Id);
             var response = await action.ExecuteAsync(CancellationToken.None);
-
             Item actual;
             response.TryGetContentValue(out actual);
 
             Assert.That(response.IsSuccessStatusCode);
-            Assert.That(actual, Is.EqualTo(expected));
+            Assert.That(actual, Is.EqualTo(expected).UsingItemComparer());
         }
 
         [Test]
-        public async Task TestGetAll()
+        public async Task Get_NoParameters_ReturnsAllItems()
         {
-            var expected = new []
+            var expected = new List<Item>
```

Comment:
It was OK when it was an array (it's shorter and more accurate code as you never add to the collection and as a bonus, it is actually smaller memory structure) – but it is such a detail that `task 2` can handle it :wink: